code/__DEFINES/movement.dm

```diff
@@ -111,3 +111,8 @@ GLOBAL_VAR_INIT(glide_size_multiplier, 1.0)
 #define TELEPORT_CHANNEL_CULT "cult"
 /// Anything else
 #define TELEPORT_CHANNEL_FREE "free"
+
+///Return values for moveloop Move()
+#define MOVELOOP_NOT_READY -1
+//FALSE 0
+//TRUE 1
```

Comment:
These should be unique defines, MOVELOOP_SUCCESS or MOVELOOP_FAILURE

Reply:
I was considering that but decided not to in the end, but i'll add it because after re-considering it its probably better to have an implicit enum